app/frontend/src/features/profile/ProfileForm.tsx

```diff
@@ -14,12 +13,28 @@ import useCurrentUser from "../userQueries/useCurrentUser";
 import useUpdateUserProfile from "./useUpdateUserProfile";
 import { useIsMounted, useSafeState } from "../../utils/hooks";
 
-const useStyles = makeStyles({
+const useStyles = makeStyles((theme) => ({
   buttonContainer: {
     display: "flex",
     paddingTop: theme.spacing(1),
   },
-});
+  field: {
+    "& > .MuiInputBase-root": {
+      width: "100%",
+    },
+    [theme.breakpoints.up("md")]: {
+      "& > .MuiInputBase-root": {
+        width: 400,
+      },
+    },
+  },
```

Comment:
```suggestion
  field: {
    "& > .MuiInputBase-root": {
      extend: 'tagInput', 
    },
  },
```
I wonder if something like that works if you lift the styles in `tagInput` above this according to the [first example on this JSS page](https://cssinjs.org/?v=v10.5.1).